### ISSUE TITLE
Add wizard for new provider setup

### ISSUE THREAD

**Issue by vbatoufflet** [facette/facette#370]
Today, the administration panel UI lacks some documentation.

To ease initial setup, it would be convenient to have a wizard-like UI when setuping new providers.

**Comment by kousu** [comment]
Yeah! https://docs.facette.io/latest/architecture/ links "providers" to https://docs.facette.io/latest/configuration/ which doesn't say _anything_ about them, and doesn't explain how I might add some with the config file. If I'm supposed to do it through UI, the UI at `/admin/sources/` doesn't use the word "Providers" anywhere, and it also doesn't seem to have an enabled Add button. What do?

**Comment by kousu** [comment]
I don't know how I feel about having a wizard, but having any way to actually use Facette would be good.
